### DISCUSSION TITLE
Add monitortags patch

### DISCUSSION THREAD

**Discussion by ghost** [bakkeby/dwm-flexipatch#195] [Ideas]
I was hoping you could add the monitortags patch. I'm a fan of how wms like xmonad manage workspaces across multiple monitors, and would love to see this functionality easily available in dwm via flexipatch. 

https://github.com/bakkeby/patches/blob/master/dwm/dwm-monitortags-6.2.diff

**Comment by bakkeby** [comment]
The concept of that patch is merely to be able to define different tag icons on a per monitor basis though.

That is also already integrated, see 
https://github.com/bakkeby/dwm-flexipatch/blob/67fc80803d09529d0edff02c5079a71387853a10/config.def.h#L379

If you are looking for actual workspaces then you may want to have a look at [dusk](https://github.com/bakkeby/dusk) instead. 

There is a patch / hack for dwm called single tagset which does something similar, but that is incompatible with practically every other patch out there so that is out of scope for dwm-flexipatch.
